debian/postinst

```diff
@@ -18,6 +18,7 @@
 case $1 in
 configure)
 	systemctl enable auditd.service
+	systemctl enable delphix-migration.service
```

Comment:
If we're going to enable this service by default, we should verify that it works without the virtualization package installed; otherwise we'll hit failures on systems that use delphix-platform but not delphix-virtualization (e.g. dc1).

I think it'll be fine, since the "perform-migration" file won't exist, but just wanted to mention it.

Reply:
I've created a `dcenter-internal` ESX image and verified that it came up.
The `delphix-migration` service was disabled at that point as this was the second boot (first boot was performed by the OVA creation tool).

```
$ sudo journalctl -u delphix-migration
sudo: unable to resolve host localhost.localdomain
-- Logs begin at Wed 2019-02-27 15:27:02 UTC, end at Wed 2019-02-27 15:47:00 UTC. --
Feb 27 15:27:03 localhost.localdomain systemd[1]: Starting Delphix OS Migration Service...
Feb 27 15:27:03 localhost.localdomain os-migration[841]: File /var/delphix/migration/perform-migration missing. Nothing to do.
Feb 27 15:27:03 localhost.localdomain systemctl[844]: Removed /etc/systemd/system/default.target.wants/delphix-migration.service.
Feb 27 15:27:03 localhost.localdomain systemd[1]: Started Delphix OS Migration Service.

$ systemctl status delphix-migration
● delphix-migration.service - Delphix OS Migration Service
   Loaded: loaded (/lib/systemd/system/delphix-migration.service; disabled; vendor preset: enabled)
   Active: inactive (dead)
```

systemctl doesn't show anything out of the ordinary:
```
$ systemctl --failed
  UNIT                     LOAD   ACTIVE SUB    DESCRIPTION
● isc-dhcp-server.service  loaded failed failed ISC DHCP IPv4 server
● isc-dhcp-server6.service loaded failed failed ISC DHCP IPv6 server
```
Those same service failures also appear in dc1.